Documentation/GeneralPrinciples/PageTree/Index.rst

```diff
@@ -124,12 +162,31 @@ page-related functions the fast way. Here is what these options do:
 - **History/Undo**: Shows the change history of the page (who did which
   changes when)
 
-- **Branch Actions**: Actions related to the whole branch
 
-  - **More options ... > Export**: Opens the export tool and preselects the selected page
+.. _new-page-drag-and-drop:
 
-  - **More options ... > Import**: Opens the import tool and preselects the selected
-    page
+Create New Pages With Drag and Drop
+===================================
+
+You can also create new pages using drag'n'drop.
+
+.. rst-class:: bignums
+
+1. Clicking on the "Create new pages" icon (top left)
+
+2. Drag and drop a "Standard" page to its location in the page tree.
+
+
+   .. figure:: ../../Images/PageTreeDragndop1.png
+      :class: with-shadow
+
+   .. figure:: ../../Images/PageTreeDragndop2.png
+      :class: with-shadow
+
+3. A new page with title [Default Title] has now been created.
```

Comment:
`A new page has now been created at the desired location. By default it will be called "[Default Title]" which can be changed right away by entering a new title.`